tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import pkg_resources
+
+import httpretty
+import pytest
+
+from inspirehep.modules.workflows.tasks.arxiv import (
+    arxiv_fulltext_download,
+    arxiv_plot_extract,
+    arxiv_refextract,
+    arxiv_author_list,
+)
+
+
+@pytest.fixture
+def arxiv_pdf():
+    return pkg_resources.resource_string(
+        __name__, os.path.join('fixtures', '0911.5233.pdf'))
+
+
+@pytest.fixture
+def another_arxiv_pdf():
+    return pkg_resources.resource_string(
+        __name__, os.path.join('fixtures', '0701031v2.pdf'))
+
+
+class StubObj(object):
+    def __init__(self, data, extra_data, files):
+        self.data = data
+        self.extra_data = extra_data
```

Comment:
For now, every test file has its own small mocks/stubs/dummies. The better approach would be to write a unique mock that covers all use cases and put it in a `conftest.py` in the same folder.